release-tool

```diff
@@ -279,7 +277,7 @@ checkGitRepository() {
 }
 
 checkReleaseDoesNotExist() {
-    if ! git tag | grep -q "^$TAG_NAME$"; then
+    if [ $(git tag -l $TAG_NAME) ]; then
```

Comment:
I think this quote-less syntax caused errors on macOS last time.
Maybe do
```suggestion
    if git rev-parse "MOE" &>/dev/null; then
```

Reply:
I just tested this on macOS and it worked fine, I much prefer the direct git tag method than the rev-parse craziness.
`if [ $(git tag -l 2.7.0-beta1) ]; then echo 1; fi`